### PR TITLE
Customized files need to be highlighted

### PR DESCRIPTION
**Pull request by yellabelly**
https://deploy-preview-1645--axway-open-docs.netlify.app/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion/#back-up-customized-files

Customized files were documented in the <7.7 update instructions and are still relevant to 7.7.*

### CODE REVIEW

**Review by maherb4 — APPROVED** [review]
The content seems like a good addition to me (seems to have a markdown error tho)

**Comment by clivequinn** [comment]
Seems good.

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md [hunk 1]
```diff
@@ -80,6 +80,22 @@ The `update_apigw.sh` script generates a trace file in the `update-output/trace`
 
 The script takes a backup of your entire API Gateway installation directory and places it in a `tar` file in the `update-output/backups` directory. Specify a different directory using the `--backup_dir` option. To manage your own backups, use the `--no_backup` option.
 
+### Customized Files
+
+Ensure to back up any customized files in your INSTALL_DIR. You should merge updated files instead of copying them back directly to avoid any regex matching issues. For example, the following directories might contain customized files:
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:85]: @yellabelly @clivequinn The instruction is not clear to me:
* You must back up your files (ok!) 
* It seems that you might update some of these files, and if you do, you should **merge** them - what does this mean?

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md [hunk 2]
```diff
@@ -80,6 +80,20 @@ The `update_apigw.sh` script generates a trace file in the `update-output/trace`
 
 The script takes a backup of your entire API Gateway installation directory and places it in a `tar` file in the `update-output/backups` directory. Specify a different directory using the `--backup_dir` option. To manage your own backups, use the `--no_backup` option.
 
+### Back up customized files
+
+Ensure to back up any customized files in your `INSTALL_DIR` directory. You must merge updated files instead of copying them back directly to avoid any regex matching issues. For example, the following directories might contain customized files:
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:85]: @yellabelly @clivequinn 

* You must back up your files (ok!)
* It seems that you might update some of the files that you've backed up, and if you do, you should merge them - what does this mean?

Comment (yellabelly) [reply content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:85]: Merge just makes sure we don't lose something that might be added to the factory versions of these custom files. If the factory ones haven't changed then copying them back is fine. Can compare the .orig files to know whether to do straight copy.
@andreamussap

**Review by andreamussap — APPROVED** [review]

**Pull request merged**